src/select/dropdown.js

```diff
@@ -68,12 +69,14 @@ export default function SelectDropDown(props: DropDownPropsT) {
           props: {
             $rows: rows,
             $type: type,
+            $itemRef: itemRef,
             $isOpen: isDropDownOpen,
             ...dropDownProps,
           },
         },
         Option: {
           props: {
+            $ref: itemRef,
```

Comment:
the ref is needed to measure height of one element in list. Yes, it could have been assigned many times, but to avoid this I will have to change inner code of Menu. The menu itself works with only one as it has only one cycle of rendering